servicebus/spring-cloud-azure-starter-integration-servicebus/single-namespace/README.md

```diff
@@ -1,183 +1,89 @@
-# Spring Cloud Azure Service Bus Integration Code Sample shared library for Java
+# Using Servicebus Queue and Topic With Spring Integration 
 
-## Key concepts
+This guide walks you through the process of accessing Servicebus Queues and Topics with Spring Integration.
 
-This code sample demonstrates how to use Spring Integration for Azure Service Bus.
+## What You Will build
+You will build an application that use Azure Servicebus Queues and Topics to send and receive messages.
 
+## What You Need
 
-## Getting started
+- [Terraform](https://www.terraform.io/)
+- [IntelliJ IDEA](https://www.jetbrains.com/idea/download/#section=mac) 
+- [Azure CLI](https://docs.microsoft.com/cli/azure/install-azure-cli)
+- JDK8
+- Maven
 
-Running this sample will be charged by Azure. You can check the usage and bill at
-[this link][azure-account].
+## Prevision Azure Resources required to run this sample.
 
+## Provision the Resources
+1. Before running terraform, there're two parameters needed to be updated in the `./terraform/variables.tf` file.
```

Comment:
terraform variables are not required to be updated in the 'tf' file only, it can be provided from commanline I think.